ride_share.rb

```diff
@@ -0,0 +1,181 @@
+# This program uses a nested data structure that holds information about 
+# rideshare drivers and rides.  It calls methods to perform calculations
+# about rides, ratings, and earnings, and displays information to the user.
+
```

Comment:
I really like how you broke down the file with the method definitions at top and the main code at the bottom! 😍